### PR TITLE
Update readme 3

### PR DESCRIPTION
**Pull request by Arminek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,21 +23,30 @@ Elastic search for Sylius.
...
+     new \Sylius\ElasticSearchPlugin\SyliusElasticSearchPlugin(),
     ```
 
 5. Create/Setup database:
 
     ```bash
-    $ app/console do:da:cr
-    $ app/console do:sch:cr
-    $ app/console syl:fix:lo
+    $ bin/console ongr:es:index:create
```
Comment (lchrusciel) [inline README.md:39]: For me, we should use full commands to make it more readable for newcomers

File: README.md [hunk 2]
```diff
@@ -23,21 +23,30 @@ Elastic search for Sylius.
...
+    $ bin/console do:da:cr
+    $ bin/console do:sch:cr
+    $ bin/console syl:fix:lo
+    ```
+
+    If there is a problem with creating elastic search index run those commands:
```
Comment (lchrusciel) [inline README.md:45]: `If there is a problem with creating elastic search index, try dropping the previous one and recreate it with those commands`?

File: README.md [hunk 3]
```diff
@@ -106,9 +115,370 @@ Elastic search for Sylius.
...
+
+```
+    /shop-api/taxons/mugs/products
+```
+
+Response:
+
+```
+{
+    "items": [
```
Comment (lchrusciel) [inline README.md:163]: Don't we have a pagination information similar to this from ShopApiPlugin?

File: tests/Responses/Expected/mugs_list_page.json [hunk 4]
```diff
@@ -234,7 +234,7 @@
                     "default": false,
                     "urlParameters": {
                         "taxon_slug": "mugs",
-                        "attributeValues": [
+                        "attribute_values": [
```
Comment (lchrusciel) [inline tests/Responses/Expected/mugs_list_page.json:237]: The response should contain camelCased keys

Comment (lchrusciel) [reply tests/Responses/Expected/mugs_list_page.json:237]: Ok, the problem is with our routeing definition... We should probably migrate it to camelCase also...

**Review by lchrusciel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pjedrzejewski) [reply README.md:39]: 👍

**Comment by lchrusciel** [comment]
@pjedrzejewski @Arminek what with my comments?

**Pull request merged**